dbtype/types.go

```diff
@@ -0,0 +1,40 @@
+// dbtype is a package that contains types used by the database driver packages for session storage.
+package dbtype
+
+import (
+	"time"
+
+	"github.com/cccteam/ccc"
+)
+
+type Session struct {
+	ID        ccc.UUID  `spanner:"Id"        db:"Id"`
+	Username  string    `spanner:"Username"  db:"Username"`
+	CreatedAt time.Time `spanner:"CreatedAt" db:"CreatedAt"`
+	UpdatedAt time.Time `spanner:"UpdatedAt" db:"UpdatedAt"`
+	Expired   bool      `spanner:"Expired"   db:"Expired"`
+}
+
+type SessionOIDC struct {
```

Comment:
Would embedding the default Session and InsertSession structs make sense here?

Reply:
Just pushed a tweak to that. LMK your thoughts.